stopNodes not in type definitions yet

<!-- ## Important 

1. If have some **questions**, please read the README or check `spec` folder for examples. **Stackoverflow** is also a good place to discuss technical questions as you get support from many people there. 
2. If you're reporting a **vulnerability** or a **security bug**, please don't expose it to the public until it is discussed with maintainer. It'll be deleted immediately. Please contact directly. You can find my contact detail in github profile.
3. Your issue may be ignored if you remove the checklist from this issue without any reason.
-->

<!-- If you are raising a **bug** please ensure to fill below checklist; -->
### Checklist
<!-- choose one by changing [ ] to [x] -->
* [ ] Have you asked your question on Stackoverflow or similar forum?
* [x] Are you running the latest version?
* [x] Have you included sample input, output, error, and expected output? 
* [x] Have you checked if you are using correct configuration?
* [ ] Did you try online [tool](https://naturalintelligence.github.io/fast-xml-parser/)? (N/A I think?)

<!-- Please provide following data to avoid long communication and fast resolution. --> 
### Input

#### Code
```typescript
import fastxml from 'fast-xml-parser';

function parse(pXMLString: string): any {
  if (fastxml.validate(pXMLString) === true) {
    return fastxml.parse(pXMLString, {
      attributeNamePrefix: '',
      ignoreAttributes: false,
      stopNodes: ['onentry', 'onexit', 'transition'],
    });
  }
  throw new Error("That doesn't look like valid xml ...\n");
}
```

### Output
When compiling the code, the typescript compiler complains stopNodes doesn't exist on `Partial<X2jOptions>`. 

(Demo repo [over here](https://github.com/sverweij/fast-xml-parser-type-issue) `npm i && npm run build` to reproduce)

Root cause seems that `X2jOptions` in [src/parser.d.ts](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/src/parser.d.ts) doesn't yet contain the stopNodes attribute that was introduced in #150.

```
src/parseit.ts:8:7 - error TS2345: Argument of type '{ attributeNamePrefix: string; ignoreAttributes: false; stopNodes: string[]; }' is not assignable to parameter of type 'Partial<X2jOptions>'.
  Object literal may only specify known properties, and 'stopNodes' does not exist in type 'Partial<X2jOptions>'.

8       stopNodes: ['onentry', 'onexit', 'transition'],
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.
```


### expected data
The code compiles without warnings.

**Would you like to work on this issue?**
<!-- choose one by changing [ ] to [x] -->
- [x] Yes
- [ ] No

--- a/src/parser.d.ts
+++ b/src/parser.d.ts
@@ -15,6 +15,7 @@ type X2jOptions = {
   parseTrueNumberOnly: boolean;
   tagValueProcessor: (tagValue: string, tagName: string) => string;
   attrValueProcessor: (attrValue: string, attrName: string) => string;
+  stopNodes: string[];
 };
 type X2jOptionsOptional = Partial<X2jOptions>;
 type validationOptions = {

